modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/ClayCriteriaBuilder.es.js

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClayCriteriaGroup from './ClayCriteriaGroup.es';
+import ClayButton from './ClayButton.es';
+
+class ClayCriteriaBuilder extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			editing: false,
+			initialCriteria: this.props.criteria
```

Comment:
Is this supposed to be used? Or is criteria supposed to be state and props is initialCriteria?

Reply:
Oh good catch...`initialCriteria` definitely should be a prop and `criteria` a state.